IFComp/root/src/_prize_list.tt

```diff
@@ -41,6 +41,9 @@ USE Markdown;
             <li>
             <p>[% IF prize.url %]<a href="[% prize.url %]">[% END %]<strong>[% prize.name | markdown | remove('</?p>') %]</strong>[% IF prize.url %]</a>[% END %]
             [% IF prize.notes %]<br />[% prize.notes | markdown | remove('</?p>') %][% END %]
+            [% IF prize.location && !prize.ships_internationally %]
+                <br /><em>This prize ships only to addresses within [% prize.location %].</em>
```

Comment:
Some of the other fields (`prize.name`, `prize.notes`) are put through the `markdown` filter. Should `prize.location` too? If not, maybe it should be put through the `html` filter?

Reply:
Yes, I suppose that everyone who wishes to limit shipments to Turks & Caicos will appreciate that.